composer.lock

```diff
@@ -3038,7 +3038,7 @@
                 "symfony/process": "v4.4.25",
                 "symfony/psr-http-message-bridge": "v2.1.0",
                 "symfony/routing": "v4.4.25",
-                "symfony/serializer": "v4.4.35",
+                "symfony/serializer": "v4.4.25",
```

Comment:
Serializer was also downgraded... I don't think composer.lock should have been updated with these changes.